.github/workflows/master_pr.yml

```diff
@@ -28,11 +28,15 @@ jobs:
         id: runner-type
         run: |
           runner=$(yq '.runner' convoy.yaml)
+          if [[ $runner == "amd-go-22" ]]; then
+            echo "Overriding the runner from ${runner} to amd"
+            runner="amd"
```

Comment:
Why only for `amd-go-22` ?

Reply:
Yes, need to include all. I used this for testing.